<launch>

  <node pkg="map_compression" type="loadProResult" name="loadProResult" output="screen">

	<param name="loadResultDir" value="/home/yh/mapModel/2018/07.19/gurobi_compress/"/>
	<param name="sectionNum" value="69"/>
	<param name="loadMapName" value="/home/yh/mapModel/2018/07.19/wholeMap_origin.vtk"/>
	<param name="saveCloudName" value="/home/yh/mapModel/2018/07.19/ILP_YQ_result.vtk"/>
	
  </node>
</launch>
